tests/next_tests/integration_tests/feature_tests/dace/test_orchestration.py

```diff
@@ -0,0 +1,203 @@
+# GT4Py - GridTools Framework
+#
+# Copyright (c) 2014-2023, ETH Zurich
+# All rights reserved.
+#
+# This file is part of the GT4Py project and the GridTools framework.
+# GT4Py is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or any later
+# version. See the LICENSE.txt file at the top-level directory of this
+# distribution for a copy of the license or check <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import numpy as np
+from typing import Optional
+from types import ModuleType
+import pytest
+
+import gt4py.next as gtx
+from gt4py.next import backend as next_backend
+
+from next_tests.integration_tests import cases
+from next_tests.integration_tests.cases import cartesian_case, unstructured_case
+from next_tests.integration_tests.feature_tests.ffront_tests.ffront_test_utils import (
+    exec_alloc_descriptor,
+    mesh_descriptor,
+    Vertex,
+    Edge,
+    E2V,
+)
+from next_tests.integration_tests.multi_feature_tests.ffront_tests.test_laplacian import (
+    lap_program,
+    laplap_program,
+    lap_ref,
+)
+
+try:
+    import dace
+    from gt4py.next.program_processors.runners.dace import run_dace_cpu, run_dace_gpu
+except ImportError:
+    dace: Optional[ModuleType] = None  # type:ignore[no-redef]
+    run_dace_cpu: Optional[next_backend.Backend] = None
+    run_dace_gpu: Optional[next_backend.Backend] = None
+
+
+@gtx.field_operator
+def _testee(a: gtx.Field[gtx.Dims[Vertex], gtx.float64]):
+    return a(E2V[0])
+
+
+@gtx.program
+def testee(a: gtx.Field[gtx.Dims[Vertex], gtx.float64], b: gtx.Field[gtx.Dims[Edge], gtx.float64]):
+    _testee(a, out=b)
```

Comment:
I think it would make sense to move these definitions closer to the actual test that is using them